### PR TITLE
3416 add logstash to front

### PR DESCRIPTION
**Pull request by oswaldquek**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :debug
+  config.log_level = :info
```
Comment (richardTowers) [inline config/environments/production.rb]: Do you need to change the comment here to reflect that the value is no longer the lowest log level?

Comment (oswaldquek) [inline config/environments/production.rb]: i think config.log_level can be deleted altogether actually. i think it logs at INFO by default in production.

Comment (richardTowers) [inline config/environments/production.rb]: I'd be inclined to keep it anyway; [explicit is better than implicit](https://www.python.org/dev/peps/pep-0020/). Maybe just remove the comment?

Comment (davbo) [inline config/environments/production.rb]: Did someone just quote the Zen of Python on our Rails project? :+1: :gem: :snake: :smile_cat:

Comment (oswaldquek) [inline config/environments/production.rb]: Amen to that :+1:

**Pull request merged**